docs: add AWS DMS to replication instructions (#40859)
supabase/supabase: apps/docs/content/guides/database/postgres/setup-replication-external.mdx

select pg_create_logical_replication_slot('example_slot', 'pgoutput');
```

> 3. Now we will connect to our **external database** and subscribe to our `publication`

Reviewer: ```suggestion
3. Now connect to your **external database** and subscribe to the `publication`
```

<Admonition type="caution">

> This will need a **direct** connection (not a Connection Pooler) to your database and you can find the connection info in the [**Connect** panel](/dashboard/project/_?showConnect=true) in the `Direct connection` section.

Reviewer: ```suggestion
This needs a **direct** connection (not a Connection Pooler) to your database and you can find the connection info in the [**Connect** panel](/dashboard/project/_?showConnect=true) in the `Direct connection` section.
```
